examples/PaymentExample.php

```diff
@@ -0,0 +1,165 @@
+<?php
+/**
+ * This file is part of the PHP Rebilly API package.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @see http://rebilly.com
+ */
+
+namespace Rebilly\Test;
```

Comment:
I'm not so sure. These examples demonstrate the possibilities and used for tests. And since PSR-4 allows you to assign two folders for one namespace, we use a separate folder for easy demonstration.